backend/src/main/java/hu/flowacademy/konyhatunder/service/ImageStorageService.java

```diff
@@ -27,7 +27,7 @@ public ImageStorageService(ImageRepository imageRepository) {
     public Image storeFile(MultipartFile file) {
         if (file == null) {
             log.debug("No image uploaded");
-            return imageRepository.findByFileName("abc").orElseThrow(() -> new MyFileNotFoundException("Nincs default image"));
+            return imageRepository.findByFileName("defaultImage").orElseThrow(() -> new MyFileNotFoundException("Nincs default image"));
```

Comment:
Kellene az a konstans, most már az is biztos, hogy public lesz.

Reply:
Rendben, az https://github.com/flowacademyhu/kappa-konyhatunder/issues/56 issuen javítva lesz ez is.